src/test/kotlin/org/jetbrains/bio/Tests.kt

```diff
@@ -22,4 +23,21 @@ object Tests {
         }
     }
 
+    fun assertEquals(expected: Double, actual: Double, precision: Double, message: String?) {
```

Comment:
Please use `Precision.equals` instead of creating own implementation.

Reply:
Nice, thanks!

Reply:
Actually, there is an `Assert.assertEquals` method with the same functionality. I couldn't find it before because it has a somewhat unexpected signature `(String, Double, Double, Double)`.